packages/client/src/runtime/core/raw-query/rawQueryArgsMapper.ts

```diff
@@ -32,19 +31,11 @@ More Information: https://pris.ly/d/execute-raw
 type RawQueryArgsMapperInput = {
   clientMethod: string
   activeProvider: string
-  driverAdapterProvider?: ErrorCapturingDriverAdapter['provider']
 }
 
 export const rawQueryArgsMapper =
-  ({ clientMethod, activeProvider, driverAdapterProvider }: RawQueryArgsMapperInput) =>
+  ({ clientMethod, activeProvider }: RawQueryArgsMapperInput) =>
   (args: RawQueryArgs) => {
-    // TODO: remove this block, probably not needed anymore
-    if (driverAdapterProvider !== undefined) {
-      // When using the driver adapter, we need to use the flavour of the adapter,
-      // in order to avoid enumerating all the possible `@prisma/*` provider names.
-      activeProvider = driverAdapterProvider
-    }
-
```

Comment:
Removed this while at it. This block was needed e.g. back when `@prisma/planetscale` was a thing.